pkg/kube/ready.go

```diff
@@ -244,6 +244,7 @@ func (c *ReadyChecker) jobReady(job *batchv1.Job) (bool, error) {
 		c.log("Job is not completed: %s/%s", job.GetNamespace(), job.GetName())
 		return false, nil
 	}
+	c.log("Job is completed: %s/%s", job.GetNamespace(), job.GetName())
```

Comment:
This still uses log, we are going to want to change this to slog. Then to label this correctly when we do the log.

Reply:
That would entail removing the log function from `func NewReadyChecker(cl kubernetes.Interface, log func(string, ...interface{}), opts ...ReadyCheckerOption) ReadyChecker {` removing c.log and calling slog instead.

Reply:
```suggestion
	slog.Info("Job is completed: %s/%s", job.GetNamespace(), job.GetName())
```

Maybe a debug, not sure

Reply:
To avoid mixing PR, I open a new one with changes on `ready.go` https://github.com/helm/helm/pull/30698